sphinxbase/default/PKGBUILD

```diff
@@ -5,23 +5,27 @@
 
 pkgname=sphinxbase
 pkgver=5prealpha
-pkgrel=11
+pkgrel=12
```

Comment:
People who already have the package do not need to rebuild it so don't increase the `pkgrel` here.